pkg/cli/debug_merge_logs.go

```diff
@@ -579,13 +581,13 @@ func seekToFirstAfterFrom(
 		var e logpb.Entry
 		d, err := log.NewEntryDecoderWithFormat(f, editMode, format)
 		if err != nil {
-			panic(err)
+			panic(errors.WithMessage(err, fmt.Sprintf("Error while processing file %s", f.Name())))
```

Comment:
nit: errors should start with a lower-case letter.

Reply:
Also, I believe you can use `errors.WithMessagef` instead of the additional `fmt.Sprintf` call.